config/software/cleanup.rb

```diff
@@ -0,0 +1,36 @@
+#
+# Copyright 2012-2016 Chef Software, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+name "cleanup"
```

Comment:
We should name this software definition something more descriptive like: `clean-static-library-files` or `clean-static-libraries`.

Why would we not add this as (optional) core functionality in Omnibus? It could operate at the end of the build much like the health check. /cc @chef/omnibus-maintainers